datafusion/core/tests/parquet/mod.rs

```diff
@@ -824,9 +863,17 @@ fn create_data_batch(scenario: Scenario) -> Vec<RecordBatch> {
         }
         Scenario::WithNullValues => {
             vec![
-                make_all_null_values(),
+                make_int_batches_with_null(5, 0, 0),
                 make_int_batches(1, 6),
-                make_all_null_values(),
+                make_int_batches_with_null(5, 0, 0),
+            ]
+        }
+        Scenario::WithNullValuesPageLevel => {
+            vec![
+                make_int_batches_with_null(5, 1, 6),
```

Comment:
I think this scenario should have at least one batch of all nulls (e.g. `make_int_batches_with_nulls(5, 0, 0)`)

Reply:
i think this will be prune by row group -> column level